### PR TITLE
Add matrix-to-irc guide

### PR DESCRIPTION
**Pull request by bakkot**

### CODE REVIEW

File: matrix-guide.md [hunk 1]
```diff
@@ -56,3 +56,15 @@
 [#tc39-beginners]: https://matrix.to/#/!OXhgybpQzCtnugpzuz:matrix.org
 [#tc39-inclusion]: https://matrix.to/#/!DgpygRnlCHLTRbahDa:matrix.org
 [#tc39-website]: https://matrix.to/#/!hmsRHUEXriRovkvcin:matrix.org
+
+
+## Coming from IRC
+
+If you're used to IRC and using the Element.io web client, you can make the transition less jarring by changing some options. You can get to the settings by clicking your username in the upper-left corner and selecting "All Settings" from the dropdown.
```
Comment (ryzokuken) [inline matrix-guide.md:63]:
```suggestion
If you're used to IRC and using the Element Web or Desktop clients, you can make the transition less jarring by changing some options. You can get to the settings by clicking your username in the upper-left corner and selecting "All Settings" from the dropdown.
```

**Review by ryzokuken — APPROVED** [review]
LGTM  overall, would you mind moving this section above the links?

**Comment by bakkot** [comment]
> would you mind moving this section above the links?

Hm, really? I feel like the links are more important, and belong in the first section.

**Comment by mpcsh** [comment]
I think Ujjwal means to move them above the list of links beginning on line 48, which are not rendered (they're like an index for resolving the references earlier in the document)

**Comment by bakkot** [comment]
Oooh, missed that. I completely forgot that markdown supports out-of-line links.

Done.

**Review by ryzokuken — APPROVED** [review]
Perfect, thanks again @bakkot and thanks @mpcsh for the clarification.

**Pull request merged**